package Custom.Exception;



public class CustomExceptionDemo
{
    public static void main(String args[]) throws Exception
    {
    	CustomExceptionDemo exceptionDemo = new CustomExceptionDemo();
        exceptionDemo.displayNumbers();
    }
 
    public void displayNumbers() throws MyException
    {
        for(int i=0;i<10;i++)
        {
            System.out.println("i= "+i);
            if(i==6)
            {
              throw new MyException("My Exception Occurred");
            }
        }
    }
}